### ISSUE TITLE
Impossible to create an empty DatabaseQueryFilter to fetch all rows from a database

### ISSUE THREAD

**Issue by AsFal** [dstotijn/go-notion#14]
#### Description
Impossible to create a DatabaseQuery with an empty DatabseQueryFilter. Fields that do not have empty values (such as embedded structs) cannot be omitted. [Source](https://www.sohamkamani.com/golang/omitempty/#values-that-cannot-be-omitted)

#### To Reproduce
Populate your environment variables with valid values for ```NOTION_SECRET_API_KEY``` and ```NOTION_DATABASE_ID``` and run the following code.

```go
package main

import (
	"context"
	"fmt"
	"os"

	"github.com/dstotijn/go-notion"
)

func main() {
	client := notion.NewClient(os.Getenv("NOTION_SECRET_API_KEY"))
	rows, err := client.QueryDatabase(context.Background(), os.Getenv("NOTION_DATABASE_ID"), &notion.DatabaseQuery{
		PageSize: 10,
	})
	fmt.Println("======= ERROR ======")
	fmt.Println(err)
	fmt.Println("======= ROWS ======")
	fmt.Println(rows)
}

```
Given the following table
<img width="874" alt="Screen Shot 2021-05-24 at 10 10 24 PM" src="https://user-images.githubusercontent.com/35551828/119429972-ed4d6a00-bcdd-11eb-990e-a76a0856ae73.png">
I go the following error
<img width="1288" alt="Screen Shot 2021-05-24 at 10 11 24 PM" src="https://user-images.githubusercontent.com/35551828/119429990-f4747800-bcdd-11eb-99d6-836c1a2083ce.png">

**Comment by maksim77** [comment]
Just specify nil instead  `&notion.DatabaseQuery`

**Comment by AsFal** [comment]
Specifying nil for ```&notion.DatabaseQuery``` wouldn't allow the use of an empty filter with custom Sort, StartCursor, or PageSize values.  This would prevent fetching all the entries from a database with more than 100 entries given the page size limit.

**Comment by dstotijn** [comment]
Fixed and released in `v0.3.2`.
